code/LINDA/LINDA_turf_tile.dm

```diff
@@ -232,17 +238,14 @@
 	archived_cycle = air_master.current_cycle
 
 /turf/simulated/proc/update_visuals()
-	var/new_overlay_type = tile_graphic()
-	if (new_overlay_type == atmos_overlay_type)
-		return
-	var/atmos_overlay = get_atmos_overlay_by_name(atmos_overlay_type)
-	if (atmos_overlay)
-		overlays -= atmos_overlay
+	overlays = 0
```

Comment:
overlays is a list, this should be either
`overlays.Cut()`
or
`overlays = list()`